utils/checks.stanza

```diff
@@ -1270,7 +1282,7 @@ public defn check-landpattern (component:JITXObject|JITXDef) :
     for side in [Top, Bottom] do :
       check check-min-copper-copper-space(lp, side)
       check check-min-copper-width(lp, side)
-      check check-min-solder-mask-sliver(layer(lp, SolderMask(side)))
+      check check-min-solder-mask-sliver(lp, layer(lp, SolderMask(side)))
```

Comment:
Passing the land pattern into `check-min-solder-mask-sliver` so its pass/fail messages have access to it.

Reply:
Awesome!